src/cljs/main/broadfcui/common/table/body.cljs

```diff
@@ -0,0 +1,122 @@
+    {:flexBasis "auto" :flexGrow 1 :flexShrink 1}
+    {:flexBasis width :flexGrow 0 :flexShrink 0}))
+
+
+(defn- header [{:keys [joined-columns sort-column sort-order set-sort style
+                       start-column-drag column-reset allow-no-sort?]}]
+  [:div {:style (merge {:display "flex"} (:row style) (:header-row style))}
+   (map-indexed
+    (fn [index {:keys [id width initial-width header visible? resizable? sortable?]}]
+      (when visible?
+        [:div {:style (merge (flex-params width)
+                             {:position "relative" :cursor (when sortable? "pointer")}
+                             (when resizable? (or (:resize-tab style)
+                                                  {:borderRight "1px solid" :marginRight -1})))}
+         [:div {:style (merge {:display "flex" :width width}
+                              (:cell style)
+                              (:header-cell style))
+                :onClick (when sortable?
+                           #(cond (not= sort-column id) (set-sort id :asc)
+                                  (= sort-order :asc) (set-sort id :desc)
+                                  allow-no-sort? (set-sort nil nil)
+                                  :else (set-sort id :asc)))}
+          [:div {:style {:flex "1 1 auto" :overflow "hidden" :textOverflow "ellipsis"}}
+           header]
+          flex/spring
+          (when (= id sort-column)
+            (icons/icon {:style {:margin "0 0.4rem 0 0.1rem"}}
+                        (if (= :asc sort-order) :sort-asc :sort-desc)))]
+         (when resizable?
+           [:div {:style {:position "absolute" :cursor "col-resize"
+                          :right -11 :top 0 :width 21 :height "100%" :zIndex 1}
+                  :onMouseDown (fn [e] (start-column-drag (utils/restructure e width index)))
+                  :onDoubleClick #(column-reset {:index index :initial-width (or initial-width 100)})}])]))
+    joined-columns)])
+
+
+(defn- body [{:keys [rows joined-columns style]}]
+  [:div {:style (:body style)}
+   (map-indexed
+    (fn [index row]
+      [:div {:style (merge {:display "flex"}
+                           (:row style)
+                           ((or (:body-row style) identity) (utils/restructure index row)))}
+       (map
+        (fn [{:keys [width visible? column-data render as-text]}]
+          (when visible?
+            (let [column-value (column-data row)
+                  rendered (render column-value)]
+              [:div {:style (merge (flex-params width) (:cell style) (:body-cell style))
+                     :title (cond as-text (as-text column-value)
+                                  (string? rendered) rendered)}
+               rendered])))
+        joined-columns)])
+    rows)])
+
+
+(defn- resolve-column-props [{:keys [initial-width as-text] :as props} behavior]
+  (merge {:resizable? (and (not= initial-width :auto) (:resizable-columns? behavior))
+          :sortable? (:sortable-columns? behavior)
```

Comment:
Column sortability overrides this global default.  Check out the library page table, the left column isn't sortable.